apis/swagger.yml

```diff
@@ -3906,6 +3906,37 @@ definitions:
       CheckpointName:
         type: "string"
 
+  ContainerCommitOptions:
+    description: "options of committing a container into a image"
+    type: "object"
+    properties:
+      Repository:
```

Comment:
Could you add more description for all of the fields?

Reply:
The pr not completed now, I will make it all done soon.

Reply:
@allencloud , add api describe in swagger.

Reply:
I think what @allencloud says is that we should add the description for each field. 

`Repository` means `Repository name for the created image` something like that